generators/client/templates/src/main/webapp/app/services/auth/_auth.service.js

```diff
@@ -145,10 +145,6 @@
         function logout () {
             AuthServerProvider.logout();
             Principal.authenticate(null);
-
-            // Reset state memory
-            delete $sessionStorage.previousStateName;
```

Comment:
Not sure how would this be different with OAuth or JWT? It's just an after login state (url), which is stored in the session storage (as opposed to local storage) and wiped right after actually logging in. 
For instance auth.session.service logout() method only wipes the authenticationToken but doesn't deal with previousStates.